src/icons/__stories__/mistica-icons-story.tsx

```diff
@@ -27,7 +27,7 @@ export default {
             control: {type: 'range', min: 24, max: 48, step: 4},
         },
     },
-    parameters: {fullScreen: true},
+    parameters: {fullScreen: true, a11y: {disable: true}},
```

Comment:
Removing the a11y addon in the icons story because it's painfully slow and makes the browser freeze.